README.md

```diff
@@ -7,8 +7,8 @@ Docker image of ROS (Robot Operating System) on Alpine Linux
 ROS2 humble is now available for Alpine 3.17 as `humble-3.17`.
 
 ### April 2023
-`melodic` (Melodic EOL on May 2023) and `noetic-3.11` (Alpine 3.11 EOL on May 2020) are dropped.
-These images are still available but no longer updated. It is highly recommended to update to `noetic-3.14` or `noetic-3.17`.
+`kinetic` (Kinetic EOL on May 2021), `melodic` (Melodic EOL on June 2023), `noetic-3.11` (Alpine 3.11 EOL on November 2021) and `noetic-3.14` (Alpine 3.14 EOL on May 2023) are dropped.
```

Comment:
This is past news and `### January 2024` should be added

Reply:
Fixed in 2a4afe896814966db79699e0545db2f7e60a8f5a. I left the changes to the `April 2023` section to mention `kinetic` but I can revert them if needed.